src/main/java/school/lemon/changerequest/java/introduction/hw2/Task7.java

```diff
@@ -2,6 +2,14 @@
 
 public class Task7 {
     public static int countEvens(int[] arr) {
-        return 0;
+        int count=0;
+        if(arr == null)
+            return 0;
+        for (int anArr : arr) {
+            if (anArr % 2 == 0)
```

Comment:
It's OK, but there is faster way to check if number is Even or Odd. Read about parity bit.